commons/ava_networks/test_gecko_network.go

```diff
@@ -65,33 +68,36 @@ func (network TestGeckoNetwork) RemoveService(serviceId networks.ServiceID) erro
 // ============= Loader Service Config ====================================
 type TestGeckoNetworkServiceConfig struct {
 	// Whether the certs used by services with this configuration will be different or not
-	varyCerts bool
+	varyCerts       bool
 	serviceLogLevel ava_services.GeckoLogLevel
 	// Used primarily for Byzantine tests but can also test heterogenous Gecko versions, for example.
 	imageName      string
 	snowQuorumSize int
 	snowSampleSize int
+	additionalCLIArgs        map[string]string // CLI Args to pass directly to Gecko
 }
 
 func NewTestGeckoNetworkServiceConfig(
-			varyCerts bool,
-			serviceLogLevel ava_services.GeckoLogLevel,
-			imageName string,
-			snowQuorumSize int,
-			snowSampleSize int) *TestGeckoNetworkServiceConfig {
+	varyCerts bool,
+	serviceLogLevel ava_services.GeckoLogLevel,
+	imageName string,
+	snowQuorumSize int,
+	snowSampleSize int,
+	additionalCLIArgs map[string]string) *TestGeckoNetworkServiceConfig {
 	return &TestGeckoNetworkServiceConfig{
 		varyCerts:       varyCerts,
 		serviceLogLevel: serviceLogLevel,
 		imageName:       imageName,
 		snowQuorumSize:  snowQuorumSize,
 		snowSampleSize:  snowSampleSize,
+		additionalCLIArgs:         additionalCLIArgs,
```

Comment:
One thought: if we expect the Byzantine image to get more and more CLI args, it might be worth creating a separate struct for adding services of the Byzantine variety to hold all the various bits (with the danger being that `additionalCliArgs` becomes a generic key-value place that all new params are dumped in). That said, I don't have strong opinions here and you guys will be owning this going forward so whatever works best for you!